install/helm/agones/values.yaml

```diff
@@ -186,6 +186,10 @@ agones:
       loadBalancerIP: ""
       loadBalancerSourceRanges: []
       annotations: {}
+      # ipFamilyPolicy: RequireDualStack
+      # ipFamilies:
+      # - IPv4
+      # - IPv6
```

Comment:
Instead of commenting out these settings, can we set them to empty values?

Reply:
What about setting them to what the default values?

```yaml
  ipFamilyPolicy: SingleStack
  ipFamilies:
  - IPv4
```